### ISSUE TITLE
Confusing error when PROJECT_BRANCH missing

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#185]
In GitLab by @ jbaudoui on Sep 13, 2016, 10:51

I forgot to specify the project_branch name, and I get the error:

```bash
Cloning https://earth.bsc.es/gitlab/cp/BSC-Cyclone-Tracker-tools.git into /esnas/autosubmit/a0ae/proj
fatal: repository 'BSC-Cyclone-Tracker-tools' does not exist
[ERROR] Can not clone https://earth.bsc.es/gitlab/cp/BSC-Cyclone-Tracker-tools.git into /esnas/autosubmit/a0ae/proj
```

which make me thought I had a problem to access the git repository.

Thanks @dmanubens  for finding the error!

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Sep 15, 2016, 11:08

Status changed to closed by commit 0c506e37ce6b39d2319d9727fab481b8302997d5

**Comment by kinow** [comment]
In GitLab by @fmassonn on Sep 26, 2016, 08:46

I have had the same with auto-nemo. I did not specify a PROJECT_BRANCH (it said it would default to master), but then got the error:

```
fmassonn@ bscearth321: /esnas/autosubmit/a0av >> autosubmit create a0av
Preparing .lock file to avoid multiple instances with same expid.

Checking configuration files...
autosubmit_a0av.conf OK
platforms_a0av.conf OK
jobs_a0av.conf OK
expdef_a0av.conf  OK
Configuration files OK

Cloning  https://earth.bsc.es/gitlab/es/auto-nemo.git into /esnas/autosubmit/a0av/proj
fatal: repository 'model' does not exist
[ERROR] Can not clone  https://earth.bsc.es/gitlab/es/auto-nemo.git into /esnas/autosubmit/a0av/proj
```
